src/main/native/CCM.c

```diff
@@ -38,16 +38,18 @@ typedef struct {
 #define THREAD_LOCAL __thread
 #endif
 
-
-// Pointers of functions that are only available on some hardware (might be null)
-PFI CCMECB; // equivalent to s390_km_native
-PFI CCMGHASH; // equivalent to s390_kimd_native
-PFI CCMzS390; // equivalent to s390_kmccm_native
-typedef size_t                          UDATA;
+typedef size_t UDATA;
 typedef signed char* arr;
 
-
-char* getVersion_CCM() {
+// Pointers of functions that are only available on some hardware (might be null)
+typedef void (*CCMECB_FuncPtr)(signed char *, signed char *, UDATA *, signed char*, UDATA *);
+typedef void (*CCMGHASH_FuncPtr)(signed char *, UDATA *, signed char*, UDATA *);
+typedef void (*CCMzS390_FuncPtr)(unsigned char*, unsigned char*, unsigned char*, long*, long *, unsigned char*, long*);
```

Comment:
Space before `*`.

Reply:
A new header file named `zHardwareFunctions.h` was created. In this file there should be a space before each `*` now including this definition.